tests/utils.py

```diff
@@ -341,13 +342,19 @@ def _application_docker_container(
     application_docker_capabilities: List[str],
     application_docker_command: Optional[List[str]] = None,
 ) -> Container:
-    container: Container = docker_client.containers.run(
+    extra_kwargs = {
+        "user": "5555:6666",
+        "mounts": application_docker_mounts,
+        "cap_add": application_docker_capabilities,
+    }
+    container: Container = start_container(
+        docker_client,
         application_docker_image,
-        detach=True,
-        user="5555:6666",
-        mounts=application_docker_mounts,
-        cap_add=application_docker_capabilities,
-        command=application_docker_command,
+        application_docker_command,
+        volumes=None,
+        privileged=True,
+        pid_mode="host",
```

Comment:
Why are you passing these 2? Previously we didn't run as privileged/pid_host when `containers.run` was used.